instance/models/server.py

```diff
@@ -232,44 +311,51 @@ def public_ip(self):
 
         return public_addr['addr']
 
-    def update_status(self, provisioning=False, rebooting=False, failed=None):
+    def update_status(self):
         """
         Refresh the status by querying the openstack server via nova
```

Comment:
@bradenmacdonald nit: is this docstring still correct? I don't think this method queries anything.